Multisig/Data/Services/Safe Claiming Service/SafeClaimingService.swift

```diff
@@ -19,6 +19,7 @@ class SafeClaimingService {
         self.url = URL(string: "https://5afe.github.io/")!
         httpClient = JSONHTTPClient(url: url, logger: logger)
         httpClient.jsonDecoder.dateDecodingStrategy = .millisecondsSince1970
+        httpClient.jsonDecoder.keyDecodingStrategy = .convertFromSnakeCase
```

Comment:
Isn't it better if we have this on the source files directly from now?

Reply:
ok, will update the data